src/gei/Commands/ReclaimMannequinCommand.cs

```diff
@@ -0,0 +1,138 @@
+            };
+
+            var forceOption = new Option("--force")
+            {
+                IsRequired = false,
+                Description = "Map the user even if it was previously mapped"
+            };
+
+
+            var verbose = new Option("--verbose")
+            {
+                IsRequired = false
+            };
+
+            AddOption(githubTargetOrgOption);
+            AddOption(mannequinUsernameOption);
+            AddOption(targetUsernameOption);
+            AddOption(forceOption);
+            AddOption(verbose);
+
+            Handler = CommandHandler.Create<string, string, string, bool, bool>(Invoke);
+        }
+
+        public async Task<bool> Invoke(
+          string githubTargetOrg,
+          string mannequinUser,
+          string targetUser,
+          bool force = false,
+          bool verbose = false)
+        {
+            const string targetApiUrl = "https://api.github.com";
+
+            _log.Verbose = verbose;
+
+            _log.LogInformation("Reclaming Mannequin...");
+
+            _log.LogInformation($"GITHUB TARGET ORG: {githubTargetOrg}");
+            _log.LogInformation($"Mannequin: {mannequinUser}");
+            _log.LogInformation($"Reclaiming User: {targetUser}");
+
+            var githubApi = _targetGithubApiFactory.Create(targetApiUrl);
+            var githubOrgId = await GetOrgId(githubApi, githubTargetOrg);
+
+            var mannequin = await githubApi.GetMannequin(githubOrgId, mannequinUser);
+
+            if (mannequin == null || mannequin.Id == null)
+            {
+                _log.LogError($"User {mannequinUser} is not a mannequin.");
+                return false;
+            }
+
+            if (mannequin.MappedUser != null && force == false)
+            {
+                _log.LogError($"User {mannequinUser} has been already mapped to {mannequin.MappedUser.Login}. Use the force option if you want to reclaim the mannequin again.");
+                return false;
+            }
+
+            var targetUserId = await githubApi.GetUserId(targetUser);
+
+            if (targetUserId == null)
+            {
+                _log.LogError($"Target user {targetUser} not found.");
+                return false;
+            }
+
+            var reclaimed = await githubApi.ReclaimMannequin(githubOrgId, mannequin.Id, targetUserId);
+
+            if (reclaimed)
+            {
+                _log.LogInformation($"{mannequinUser} ({mannequin.Id}) mapped to {targetUser} ({targetUserId})");
+            }
+            else
+            {
+                _log.LogInformation($"Failed to map {mannequinUser} ({mannequin.Id}) to {targetUser} ({targetUserId})");
+            }
+
+            return reclaimed;
+        }
+
+        private async Task<string> GetOrgId(GithubApi github, string githubOrg)
+        {
+            if (github == null)
+            {
+                throw new ArgumentException("Argument cannot be null", nameof(github));
```

Comment:
For `null` args `ArgumentNullException` should be thrown instead of `ArgumentException`:
```suggestion
                throw new ArgumentNullException(nameof(github));
```

Reply:
Fixed [e9b270](https://github.com/github/gh-gei/pull/271/commits/e9b27054a8ff9470a49b85b3a02210190269b3b0)